savant/app/get_scanned.py

```diff
@@ -21,18 +21,19 @@ def parse(scanner_directory):
         parser.parse()
 
 
-def get_items(filename):
-    if str(filename.filename).__contains__("debs"):
-        get_debs(filename)
+def get_items(filenames):
+    for filename in filenames:
+        if str(filename.filename).__contains__("debs"):
```

Comment:
Please use `in` here instead of `__contains__`. The "double-underscore" operators are meant to be called only internally. See https://stackoverflow.com/questions/1964934/what-does-contains-do-what-can-call-contains-function.